lib/osf-components/addon/components/editable-field/provider-metadata-manager/template.hbs

```diff
@@ -0,0 +1,18 @@
+<div
+    {{did-insert this.setCurrentProviderMetadata this.registration}}
+>
+    {{yield (hash
+        save=(perform this.save)
+        cancel=(action this.cancel)
+        startEditing=(action this.startEditing)
+        inEditMode=this.inEditMode
+        userCanEdit=this.userCanEdit
+        shouldShowField=this.shouldShowField
+        currentProviderMetadata=this.currentProviderMetadata
+        providerMetadata=@registration.providerSpecificMetadata
+        setCurrentProviderMetatdata=this.setCurrentProviderMetadata
```

Comment:
`setCurrentProviderMetatdata` does not need to be exposed since it's only used here (if it's needed at all)
```suggestion
        
```